README.md

```diff
@@ -65,6 +65,40 @@ High performance and convenient features are the main goals.
     cargo install --locked --git https://github.com/pamburus/hl.git
     ```
 
+* Run by using [nix](https://nixos.org/download/)
```

Comment:
Could you please also add an example how to install it using [nix profile](https://nix.dev/manual/nix/2.22/command-ref/new-cli/nix3-profile-install)?

Reply:
I don't have any experience with `nix profile`.

Reply:
The syntax is the same as `nix run`: `nix profile install github:pamburus/hl`